src/app/places/place-item/place-item.component.html

```diff
@@ -14,7 +14,7 @@
     <md-list>
       <md-list-item>
         <h4 md-line>Adreça:</h4>
-        <p md-line>{{place?.address}} {{place?.zip}} {{place?.city}}</p>
+        <p md-line class="no-overflow">{{place?.address}} {{place?.zip}} {{place?.city}}</p>
```

Comment:
The rulset defined there won't apply because of the specificity added by `.mat-line`.